internal/customtypes/headers.go

```diff
@@ -0,0 +1,92 @@
+// Copyright © 2025 Ping Identity Corporation
+
+package customtypes
+
+import (
+	"fmt"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/spf13/pflag"
+)
+
+var headers []string
+
+type Header struct {
+	Key   string
+	Value string
+}
+
+type HeaderSlice []Header
+
+// Verify that the custom type satisfies the pflag.Value interface
+var _ pflag.Value = (*HeaderSlice)(nil)
+
+func IsValidHeader(header string) (Header, bool) {
```

Comment:
`func IsValidHeader(header string)` -> `func (h Header) IsValid()`

Reply:
nvm I see this function parses a string instead. Perhaps a different function name then